.travis.yml

```diff
@@ -1,18 +1,27 @@
 language: php
 
 php:
-  - 5.4
   - 5.5
   - 5.6
   - 7
 
+env:
+   - SYMFONY_VERSION=2.3.*
+   - SYMFONY_VERSION=2.7.*
+   - SYMFONY_VERSION=2.8.*
+   - SYMFONY_VERSION=3.0.*
+      
 branches:
   only:
     - master
 
 before_script:
   - wget http://getcomposer.org/composer.phar
-  - php composer.phar install --dev
+  - php composer.phar require symfony/dependency-injection:${SYMFONY_VERSION} --no-update --dev
+  - php composer.phar require symfony/event-dispatcher:${SYMFONY_VERSION} --no-update --dev
+  - php composer.phar require symfony/config:${SYMFONY_VERSION} --no-update --dev
+  - php composer.phar require symfony/yaml:${SYMFONY_VERSION} --no-update --dev
```

Comment:
IMO, we should check at least against different version of `symfony/dependency-injection`. 
This fix concerns the compatibility with version 3 of this component.
I can remove this, not a problem for me